README.md

```diff
@@ -414,6 +414,8 @@ service deployed on AWS, GCP, and other platforms in the future. The specified
 origin would need to be on an allowlist maintained by the browser. If none is
 specified, a default will be used.
 
+This allowlist matches the Attribution Reporting API's, available [here](https://github.com/WICG/attribution-reporting-api/blob/main/aggregation_coordinator_origin_allowlist.md).
```

Comment:
very minor: could we add a new line between `available` and `[here](...)`? (This won't affect the rendered document, but makes the plaintext a bit easier to read (with shorter lines)

Reply:
Done, thanks for the guidance!

Reply:
Great, thanks!